lib/resty/cassandra/cluster.lua

```diff
@@ -550,6 +551,15 @@ function _Cluster:refresh()
   -- initiate the load balancing policy
   self.lb_policy:init(peers)
 
+  if self.stream_ids == nil then
+    -- Initialize the list of available seed ids (only once)
+    self.stream_ids = {}
```

Comment:
No, inserting at the beginning of an array as I'm doing takes O(N) but according to the docs is implemented in C and it's very fast.

I run get_stream_id and release_stream_id in a loop 10,000 times and I saw that each call took on average:
get_stream_id: 2.4 usec
release_stream_id: 54usec

Another possibility is to use a linked list instead of an array, which has the same features as the current implementation, O(1) get and put but uses a bit more memory.

I'm not completely against the counter implementation though since we only use the v3 protocol so it should work fine for us, less for people on v2